schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py

```diff
@@ -427,6 +455,8 @@ async def lookup_emdb(emdb_id: str) -> Tuple[str, bool]:
 
 @alru_cache
 async def lookup_pdb(pdb_id: str) -> Tuple[str, bool]:
+    # Strip the PDB- prefix
```

Comment:
nit: I don't think this comment adds any additional clarity to the code](nit: I don't think this comment adds any additional clarity to the code. I suggest removing.)